package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.57",
```

Comment:
@mlh-dungpq cc: @mlh-binhnt @themitvp 
`@material-ui/lab ` still in alpha version. is this often used in ML's production projects?
https://www.npmjs.com/package/@material-ui/lab
![image](https://user-images.githubusercontent.com/16083322/103508817-18592d80-4ea5-11eb-9fd1-404146ee260c.png)

Reply:
i removed it